Extensions/xkit_preferences.js

```diff
@@ -19,6 +20,93 @@ XKit.extensions.xkit_preferences = new Object({
 
 	hide_xcloud_if_not_installed: false,
 
+	dark_mode: `:root {
```

Comment:
to keep the indentation more sane, what we usually do for large strings goes like

```js
big_string_property: `
	.someClass {
		/* css rules */
	}
`,
```

so that the closing quote and ending comma have the same indentation as the property name, the same way functions have their closing brace and ending comma indented the same as their property names.